### PR TITLE
Resend docs and expiration date

### PR DESCRIPTION
**Pull request by ScottyGreenburg**

### CODE REVIEW

**Comment by TangoJim** [comment]
I would add "if applicable". Otherwise, looks good to me.

"For international variable skus, there will be an expiration date ("expiration") in the successful response, if applicable."

File: README.md [hunk 1]
```diff
@@ -483,14 +484,125 @@ Example request/response:
...
+      				"token"     : "55b95ad97f5b99.96804739",
+      				"number"    : "7001204001834392312",
+      				"pin"       : "861693"
+    			}
+		}
+	}
+
+For international variable skus, there will be an expiration date ("expiration") in the successful response. The format for expiration date is ISO6801 standard date format, example: 2016-06-19 (Year-Month-Day).
+
+*Expiration dates in email templates will appear in localized format, for example: 2016-06-19 could be 2016年6月19日日曜日. We are using this PHP library to format international dates: http://php.net/manual/en/class.intldateformatter.php
```
Comment (cheklim) [inline README.md]: The last three Chinese characters (日曜日) are erroneous and should be removed: "2016-06-19 could be 2016年6月19日日曜日"

File: README.md [hunk 2]
```diff
@@ -483,14 +484,125 @@ Example request/response:
...
+Format:
+
+POST raas/v1/orders/{ORDER_NUMBER}/resend
+
+RULES & IMPORTANT NOTES
+1.	Resend is not supported for emails originally sent prior to February 18, 2015
+2.	The target order (to be resent) must have been sent with the "send_reward" property set to "true" (the default value). This means Tango Card sent the original email. If it was “false” we cannot resend the email because we never sent the original. 
+3.	Orders can be resent once every 24 hours. This is reflected in the error message timestamp if a second attempt is made less than 24 hours after the last send.
+4.	Resend functionality occurs at the platform level of the RaaS integration. 
+5.	You will need the original order number to in order to perform the resend. The Tango Card Order History call allows for a search of past orders and includes the order number for order result.
```
Comment (cheklim) [inline README.md]: An extra 'to' in 'You will need the original order number to in order to perform the resend'.

File: README.md [hunk 3]
```diff
@@ -483,14 +484,125 @@ Example request/response:
...
+
+POST raas/v1/orders/{ORDER_NUMBER}/resend
+
+RULES & IMPORTANT NOTES
+1.	Resend is not supported for emails originally sent prior to February 18, 2015
+2.	The target order (to be resent) must have been sent with the "send_reward" property set to "true" (the default value). This means Tango Card sent the original email. If it was “false” we cannot resend the email because we never sent the original. 
+3.	Orders can be resent once every 24 hours. This is reflected in the error message timestamp if a second attempt is made less than 24 hours after the last send.
+4.	Resend functionality occurs at the platform level of the RaaS integration. 
+5.	You will need the original order number to in order to perform the resend. The Tango Card Order History call allows for a search of past orders and includes the order number for order result. 
+6.	Orders can only be resent to the original recipient. If an administrator needs to verify reward information for a given order, consider using the Get Order Information call instead of RESEND.
```
Comment (cheklim) [inline README.md]: Any reason why RESEND is all caps?

File: README.md [hunk 4]
```diff
@@ -483,14 +484,125 @@ Example request/response:
...
+    	< Content-Type: application/json; charset=utf-8
+    	<
+    	{
+        	"success": true
+    	}
+
+Error Reponses:
+
+Cannot send more than once every 24 hours
+Resend functionality is limited to once every 24 hours per order. UTC time frame.
```
Comment (cheklim) [inline README.md]: 'UTC time frame' needs a bit more explanation. Last sent time reported in UTC.

File: README.md [hunk 5]
```diff
@@ -483,14 +484,125 @@ Example request/response:
...
 
+Format:
+
+POST raas/v1/orders/{ORDER_NUMBER}/resend
+
+RULES & IMPORTANT NOTES
+1.	Resend is not supported for emails originally sent prior to February 18, 2015
+2.	The target order (to be resent) must have been sent with the "send_reward" property set to "true" (the default value). This means Tango Card sent the original email. If it was “false” we cannot resend the email because we never sent the original. 
+3.	Orders can be resent once every 24 hours. This is reflected in the error message timestamp if a second attempt is made less than 24 hours after the last send.
+4.	Resend functionality occurs at the platform level of the RaaS integration.
```
Comment (cheklim) [inline README.md]: What does 'happen at the platform level' mean?

File: README.md [hunk 6]
```diff
@@ -483,14 +484,125 @@ Example request/response:
...
+2.	The target order (to be resent) must have been sent with the "send_reward" property set to "true" (the default value). This means Tango Card sent the original email. If it was “false” we cannot resend the email because we never sent the original. 
+3.	Orders can be resent once every 24 hours. This is reflected in the error message timestamp if a second attempt is made less than 24 hours after the last send.
+4.	Resend functionality occurs at the platform level of the RaaS integration. 
+5.	You will need the original order number to in order to perform the resend. The Tango Card Order History call allows for a search of past orders and includes the order number for order result. 
+6.	Orders can only be resent to the original recipient. If an administrator needs to verify reward information for a given order, consider using the Get Order Information call instead of RESEND.
+7.	Resending an email does not guarantee delivery. If the recipient does not receive an email due to spam filter settings, corporate firewalls, etc. resending an email will not get them their reward. The underlying problem will need to be determined and resolved.
```
Comment (cheklim) [inline README.md]: I would remove 'Resending an email does not guarantee delivery.' The rest of the paragraph explains it clearly.

**Pull request closed without merging**